src/main/java/homework_7/KittenToCatFunction.java

```diff
@@ -0,0 +1,6 @@
+package homework_7;
+
+@FunctionalInterface
+public interface KittenToCatFunction<Kitten> {
+     Cat grow(Kitten kitten);
+}
```

Comment:
Opt: Usually generics are letters, like K or C. Also I don't understand why you have only one generic - for my understanding, here should be 2 or none

Reply:
I think I should go through that topic again